pom.xml

```diff
@@ -202,6 +202,7 @@
         <clj-time.version>0.8.0</clj-time.version>
         <curator.version>2.9.0</curator.version>
         <json-simple.version>1.1</json-simple.version>
+        <msgpack.version>0.6.12</msgpack.version>
```

Comment:
Annoyingly, they changed the whole API with the change from `msgpack` to `msgpack-core`, so the template approach won't work anymore.  You can see new usage examples [here](https://github.com/msgpack/msgpack-java/blob/develop/msgpack-core/src/test/java/org/msgpack/core/example/MessagePackExample.java).